pkg/info/info.go

```diff
@@ -22,14 +25,51 @@ const (
 	// GitHub README page
 	UpstreamREADMETemplate = "https://github.com/openshift/backplane-cli/-/blob/%s/README.md"
 
+	VersionAddressTemplate = "https://github.com/openshift/backplane-cli/releases/download/v%s/backplane-cli_%s_%s_%s.tar.gz" // version, version, GOOS, GOARCH
+
 	// GitHub Host
 	GitHubHost = "github.com"
 )
 
 var (
-	// Version of the backplane-cli
-	// This will be set via Goreleaser during the build process
-	Version string
 
-	UpstreamREADMETagged = fmt.Sprintf(UpstreamREADMETemplate, Version)
+	GitCommit string
+
+
+	Version string
 )
+
+
+type gitHubResponse struct {
+	TagName string `json:"tag_name"`
+}
+
+
+func GetLatestVersion() (latest string, err error) {
```

Comment:
This function is already available on 
https://github.com/openshift/backplane-cli/blob/d0c230a4d9521734faa6f898c8da4faa9d65e746/internal/github/github.go#L40